dali/CMakeLists.txt

```diff
@@ -47,16 +49,25 @@ if(${CMAKE_VERSION} VERSION_LESS "3.9.0")
   # So this is a workaround to append PRIVATE in front of it 
   set(CUDA_LIBRARIES ${CUDA_LINK_LIBRARIES_KEYWORD} ${CUDA_LIBRARIES})
 endif()
+cuda_add_library(${dali_kernel_lib} STATIC ${DALI_KERNEL_SRCS})
+
 cuda_add_library(${dali_lib} SHARED ${DALI_SRCS} ${DALI_PROTO_OBJ})
 
 # Options for using Dockerfile FFmpeg version
 target_compile_definitions(${dali_lib} PRIVATE HAVE_AVSTREAM_CODECPAR=1)
 target_compile_definitions(${dali_lib} PRIVATE HAVE_AVBSFCONTEXT=1)
 
+target_compile_definitions(${dali_kernel_lib} PRIVATE HAVE_AVSTREAM_CODECPAR=1)
+target_compile_definitions(${dali_kernel_lib} PRIVATE HAVE_AVBSFCONTEXT=1)
```

Comment:
Why not "dali_operator_lib"?
My point is: operators and kernels are very different beings; they operate on different data types and follow quite different logic. Perhaps when we finally have the kernels as a separate entity, we'd add `dali_kernel_lib`.

Reply:
Done